imgee/utils.py

```diff
@@ -4,15 +4,16 @@
 import os.path
 from subprocess import check_output, CalledProcessError
 from urlparse import urljoin
-from uuid import uuid4
 
 from boto import connect_s3
 from boto.s3.bucket import Bucket
 from boto.s3.key import Key
 import defusedxml.cElementTree as elementtree
 from flask import request
 import magic
```

Comment:
If `magic` is a Python standard lib, it should be imported above, before third party libs.